samples/py_custom_module/decode_secret_message.py

```diff
@@ -96,17 +96,17 @@ def jointokens(self, token_list_ref):
 
 
 def compile():
-  vmfb_contents = compiler.tools.compile_file(os.path.join(
-      os.path.dirname(__file__), "main.mlir"),
-                                              target_backends=["vmvx"])
-  return rt.VmModule.from_flatbuffer(vmfb_contents)
+  return compiler.tools.compile_file(os.path.join(os.path.dirname(__file__),
+                                                  "main.mlir"),
+                                     target_backends=["vmvx"])
 
 
 def main():
   print("Compiling...")
-  main_module = compile()
+  vmfb_contents = compile()
```

Comment:
Note: I need to flip this around and rework the JIT APIs so this is fine for now.